tools/sklearn/search_model_validation.xml

```diff
@@ -3,7 +3,9 @@
     <macros>
         <import>main_macros.xml</import>
     </macros>
-    <expand macro="python_requirements"/>
+    <expand macro="python_requirements">
+        <requirement type="package" version="0.6">skrebate</requirement>
```

Comment:
We should add https://arxiv.org/abs/1711.08477 to the citations of the tools that use it.

Reply:
Good comment. Thanks for catching it and merging the PR.